pkg/resourcemanager/config/env.go

```diff
@@ -110,10 +125,20 @@ func GetExpectedConfigurationVariables() []ConfigRequirementType {
 }
 
 func ParseBoolFromEnvironment(variable string) bool {
-	value, err := strconv.ParseBool(envy.Get(variable, "0"))
+	env := envy.Get(variable, "0")
+	value, err := strconv.ParseBool(env)
 	if err != nil {
-		log.Printf("WARNING: invalid input value specified for bool %v: \"%v\", disabling\n", variable, value)
+		log.Printf("WARNING: invalid input value specified for %q, expected bool, actual: \"%v\". Disabling\n", variable, env)
```

Comment:
Can you use %q to provide the quotes, given that `env` is a string?

Reply:
Yeah good suggestion - done.